### ISSUE TITLE
Nodes shown are limited to 50

### ISSUE THREAD

**Issue by gmachtel** [threefoldtech/tfchain_explorer#42]
It looks like number of nodes shown is limited to 50, whereas more nodes are registered on the blockchain, on testnet (https://explorer.test.grid.tf/). 

Either a UI issue or a graphql problem.

**Comment by xmonader** [comment]
on graphql 
```
query MyQuery {
  nodes {
    id
  }
}

```

result is

```
{
  "data": {
    "nodes": [
      {
        "id": "ZIl6Ysm3dKg"
      },
      {
        "id": "m2Ce3Axs8B"
      },
      {
        "id": "yDFuWNlsoXi"
      },
      {
        "id": "2VHNsrnv3y"
      },
      {
        "id": "ty4Ej6Weo5e"
      },
      {
        "id": "v533TC0Ess"
      },
      {
        "id": "6JvpvlIB_j"
      },
      {
        "id": "uxlPcT-xJ1T"
      },
      {
        "id": "AZkRVAWDFR"
      },
      {
        "id": "jfYKFboS8F"
      },
      {
        "id": "1o8K6PUkSB3"
      },
      {
        "id": "BIKxmcHwe6"
      },
      {
        "id": "MKiGI-q5Vc"
      },
      {
        "id": "ENe606zQyL"
      },
      {
        "id": "ne-_BAh3FP"
      },
      {
        "id": "Ww2TNry9XM"
      },
      {
        "id": "hoJwdeAFcYA"
      },
      {
        "id": "P7_4ClQoui"
      },
      {
        "id": "HmDt9Ao7QLq"
      },
      {
        "id": "sjD2Ee4_3w"
      },
      {
        "id": "FIQNBdx2Esp"
      },
      {
        "id": "4RomK0LQk-d"
      },
      {
        "id": "N_WBAH6KDC"
      },
      {
        "id": "iR0SyrQcU_"
      },
      {
        "id": "Wpi_S27ke2"
      },
      {
        "id": "3HuCXg7dcI"
      },
      {
        "id": "JdAnkQwCKa"
      },
      {
        "id": "UsyWoY_IZ8"
      },
      {
        "id": "SfNsgpTySsY"
      },
      {
        "id": "K6UWKiFYty"
      },
      {
        "id": "Nr4gFJ53HS"
      },
      {
        "id": "TSsL1qRe6p"
      },
      {
        "id": "FL9OHKDKRU"
      },
      {
        "id": "gF6v1NYu0b"
      },
      {
        "id": "2z5OH6Yslj"
      },
      {
        "id": "kI5JCKjZm5"
      },
      {
        "id": "g7So9qkys7e"
      },
      {
        "id": "vsMGoHlPuq"
      },
      {
        "id": "Mu9L6xjzpe"
      },
      {
        "id": "TBW6LPO7rk"
      },
      {
        "id": "9z8DM6sQz1"
      },
      {
        "id": "VxwE9P_9AQW"
      },
      {
        "id": "geZxvzm9df"
      },
      {
        "id": "nQubjC1FS5g"
      },
      {
        "id": "4vGqDumasz"
      },
      {
        "id": "fDtPgtbtJl"
      },
      {
        "id": "Ota5YQ5MTS"
      },
      {
        "id": "A-7v5DE4I7"
      },
      {
        "id": "V2BWRbC4hCd"
      },
      {
        "id": "q56uIxN2JI"
      }
    ]
  }
}
```
50 nodes. @DylanVerstraete is their some sort of pagination or that's all the data?

**Comment by xmonader** [comment]
Fixed in #54
